drivers/gpu/drm/virtio/virtgpu_display.c

```diff
@@ -55,8 +55,9 @@ static int virtio_irq_enable_vblank(struct drm_crtc *crtc)
 	struct virtio_gpu_device *vgdev = dev->dev_private;
 	struct virtio_gpu_output *output = drm_crtc_to_virtio_gpu_output(crtc);
 
-	virtio_gpu_vblank_poll_arm(vgdev->vblank[output->index].vblank.vq);
-	virtqueue_enable_cb(vgdev->vblank[output->index].vblank.vq);
+	do {
+		virtio_gpu_vblank_poll_arm(vgdev->vblank[output->index].vblank.vq);
+	} while (!virtqueue_enable_cb(vgdev->vblank[output->index].vblank.vq));
```

Comment:
In which case, virtqueue_enable_cb will return false and need run multiple times, then success?

Reply:
robust enhance to avoid dead loop.

Reply:
Should be safe enough to use like this. We can see the same usage in virtio-GPU.